pytorch-dev.yaml

```diff
@@ -35,6 +35,8 @@ dependencies:
   - fsspec
   - ccache
   - sysroot_linux-64>=2.17
+  # Needed to find cuda by triton
+  - conda-gcc-specs
```

Comment:
I installed this in my environment on qgpu3 but still get the same error.

Reply:
What's the version and build number?

Reply:
`conda list -n pytorch-dev | grep conda-gcc-specs`

Reply:
@isuruf I get
```
conda-gcc-specs           11.4.0               h240829a_5    conda-forge
```

Reply:
That should work. You are using conda provided gcc right?

Reply:
Yup
```
(pytorch-dev) pbell@qgpu3:~/git/pytorch$ conda list | grep "g[cx]"
_libgcc_mutex             0.1                 conda_forge    conda-forge
conda-gcc-specs           11.4.0               h240829a_5    conda-forge
gcc                       11.4.0               h7baecda_2    conda-forge
gcc_impl_linux-64         11.4.0               h7aa1c59_3    conda-forge
gxx                       11.4.0               h7baecda_2    conda-forge
gxx_impl_linux-64         11.4.0               h7aa1c59_3    conda-forge
libgcc                    7.2.0                h69d50b8_2    conda-forge
libgcc-devel_linux-64     11.4.0             h922705a_103    conda-forge
libgcc-ng                 13.2.0               h807b86a_3    conda-forge
```

Reply:
Yeah he actually did this already. Turns out I needed the `cuda-driver-dev` package from conda-forge.

Reply:
Does this yaml file also need that package?

Reply:
Looks like the cuda install from `conda-forge` pulls it in automatically but it may be worth adding explicitly with a comment saying to keep it even if using system cuda installs.

Reply:
By "cuda from conda-forge" do you mean the cuda drivers? Can we do that? If so, it'd be good to add it (perhaps commented) to the file.
If you mean just the cuda-toolkit, I don't think that's the case, as I tried this yaml as-is today and I got the same error...

Reply:
If you leave out https://github.com/Quansight/torch-build/blob/23dee531bc157f9e73f01f43f9f2c337c32be74e/pytorch-dev.yaml#L44-L51, you'll run into this problem again. The fix is to keep https://github.com/Quansight/torch-build/pull/9 even if you don't install cuda toolkit from conda-forge.

Reply:
my point is that, this time, I did install all these from conda-forge. I would assume that, by Peter's comment above, I should already get `cuda-drivers-dev`, but I guess I didn't?

Reply:
Yeah, https://github.com/Quansight/torch-build/pull/9 should fix things and not rely on implicit behaviour.